Solutions/OneLoginIAM/Data Connectors/OneLoginIAMLogs_ccp/OneLoginIAMLogs_DCR.json

```diff
@@ -0,0 +1,167 @@
+                            "type": "real"
+                        },
+                        {
+                            "name": "app_name",
+                            "type": "string"
+                        },
+                        {
+                            "name": "app_id",
+                            "type": "real"
+                        },
+                        {
+                            "name": "role_name",
+                            "type": "string"
+                        },
+                        {
+                            "name": "role_id",
+                            "type": "real"
+                        },
+                        {
+                            "name": "policy_name",
+                            "type": "string"
+                        },
+                        {
+                            "name": "policy_id",
+                            "type": "real"
+                        },
+                        {
+                            "name": "policy_type",
+                            "type": "string"
+                        },
+                        {
+                            "name": "user_id",
+                            "type": "real"
+                        },
+                        {
+                            "name": "user_name",
+                            "type": "string"
+                        },
+                        {
+                            "name": "actor_system",
+                            "type": "string"
+                        },
+                        {
+                            "name": "custom_message",
+                            "type": "string"
+                        },
+                        {
+                            "name": "account_id",
+                            "type": "real"
+                        },
+                        {
+                            "name": "ipaddr",
+                            "type": "string"
+                        },
+                        {
+                            "name": "actor_user_name",
+                            "type": "string"
+                        },
+                        {
+                            "name": "actor_user_id",
+                            "type": "real"
+                        },
+                        {
+                            "name": "notes",
+                            "type": "string"
+                        },
+                        {
+                            "name": "event_type_id",
+                            "type": "real"
+                        },
+                        {
+                            "name": "created_at",
+                            "type": "string"
+                        }
+                    ]
+                },
+                "Custom-OneLoginUsersV2_CL": {
+                    "columns": [
+                        {
+                            "name": "TimeGenerated",
```

Comment:
are you getting this field "TimeGenerated" in response ? if not kindly remove it